tests/integration/ha/test_ha.py

```diff
@@ -261,6 +264,7 @@ async def test_kill_db_process_node_with_elected_cm(
     await assert_continuous_writes_consistency(ops_test, c_writes, app)
 
 
+@pytest.mark.skip
```

Comment:
hm, maybe tests skipped by mistake

Reply:
I added these to make sure I'm running the tests I've added, to guarantee they're running ok. These markers will be removed before merge.